src/frontend/src/eth/components/send/SendModal.svelte

```diff
@@ -1,7 +1,8 @@
 <script lang="ts">
 	import SendTokenModal from '$eth/components/send/SendTokenModal.svelte';
 	import SendTokenContext from '$eth/components/send/SendTokenContext.svelte';
-	import { token } from '$lib/derived/token.derived';
+
```

Comment:
It is a small thing, but may you please remove this empty paragraph?

Reply:
Definitely, good catch!